package.json

```diff
@@ -6,6 +6,7 @@
     "start": "ts-node src/index.ts",
     "test": "mocha --require ts-node/register --recursive ./src/**/*.test.ts",
     "test-watch": "npm test -- -b --watch --watch-files src",
+    "install": "docker-compose -f docker-compose.dev.yml run --rm dev npm i",
```

Comment:
In order to install the packages using a docker-compose, you can use `docker-compose -f docker-compose.test.yml run --rm test npm i`. There's no need for a new compose file.